.github/workflows/conventional-pr-check.yml

```diff
@@ -8,7 +8,7 @@ on:
       - synchronize
 
 jobs:
-  main:
+  doc-labels:
```

Comment:
Looks like github didn't run it which makes sense since it's a new workflow. I think it only runs workflows in main (but uses the workflow file from the PR)

As long as we can still merge this PR without the check passing/failing then we should be good.

We should cut a release (using the old style) just before or after merging this

Reply:
Yes, what I'll do is cut a release, update the required checks, then merge this. All open PRs will need to match.